test/recipes/95-test_external_oqsprovider_data/oqsprovider.sh

```diff
@@ -33,6 +33,11 @@ export PATH="$O_EXE:$PATH"
 export LD_LIBRARY_PATH="$O_LIB:$LD_LIBRARY_PATH"
 export OPENSSL_ROOT_DIR="$O_LIB"
 
+# Temporarily override IANA ML-KEM TLS group codepoints
+export OQS_CODEPOINT_FRODO640AES=65024
+export OQS_CODEPOINT_FRODO640SHAKE=65025
+export OQS_CODEPOINT_FRODO976AES=65026
+
```

Comment:
??? Why are we overriding FRODO codepoints?

Reply:
Because they are not assigned and the OQS assignment from 2018 was in the wrong number space: They were -- what turned out in 2024, ML-KEM code points :-)

Reply:
But what is missing here is the correct assignment of the IANA code points for ML-KEM: The previously released oqsprovider still carries old code points. The proper "quick fix" (also triggering the known handshake test & error) would be in https://github.com/vdukhovni/openssl/pull/1.

Reply:
Is there any difficulty from the fact that the oqsprovider offers the *same* group names (under different code points). Does that cause any confusion? I've pushed the change. We'll see what happens.

Reply:
But if we're assigning the correct ML-KEM code points, we don't need to bother fixing the Frodo assignments perhaps?

Reply:
Well, if we don't re-assign the wrong Frodo code points we would have _3_ (three!) KEM algorithms triggering on the same code points: frodo and mlkem from oqs and ML-KEM from default provider. I'm not sure we want _that_ (cool test, though -- that _I_ don't want to debug... :-)